core/src/main/scala/org/apache/spark/h2o/converters/H2ORDDLike.scala

```diff
@@ -23,7 +23,8 @@ import water.fvec.{Frame, FrameUtils}
 /**
  * Contains functions that are shared between all H2ORDD types (i.e., Scala, Java)
  */
-private[converters] trait H2ORDDLike[T <: Frame] {
```

Comment:
You mean, instead of private[converters]?

Thanks,
-Vlad

On Thu, Aug 25, 2016 at 10:27 AM, Michal Malohlava <notifications@github.com
> wrote:

> In core/src/main/scala/org/apache/spark/h2o/converters/H2ORDDLike.scala
> <https://github.com/h2oai/sparkling-water/pull/58#discussion_r76288004>:
>
> > @@ -23,7 +23,8 @@ import water.fvec.{Frame, FrameUtils}
> >  /**
> >   * Contains functions that are shared between all H2ORDD types (i.e., Scala, Java)
> >   */
> > -private[converters] trait H2ORDDLike[T <: Frame] {
>
> can we do private[spark] it is fine for me to limit visibility in this
> way.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/h2oai/sparkling-water/pull/58/files/dda8bacb29caf9d9ad724922e0bdf29e831b7ecb#r76288004>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAOuK2uIahFBjoJl5dSIyzjDUMlfSb4Kks5qjdCGgaJpZM4Jsgi_>
> .
>